src/Practice2_Animal/Animal.java

```diff
@@ -0,0 +1,43 @@
+package Practice2_Animal;
+
+public class Animal {
+	protected String ten;
+	protected int tuoi;
+	protected String giong;
+
+	public String getTen() {
+		return ten;
+	}
+
+	public void setTen(String ten) {
+		this.ten = ten;
+	}
+
+	public int getTuoi() {
+		return tuoi;
+	}
+
+	public void setTuoi(int tuoi) {
+		this.tuoi = tuoi;
+	}
+
+	public String getGiong() {
+		return giong;
+	}
+
+	public void setGiong(String giong) {
+		this.giong = giong;
+	}
+
+	public Animal(String ten, int tuoi, String giong) {
+		super();
+		this.ten = ten;
+		this.tuoi = tuoi;
+		this.giong = giong;
+	}
+	
+	public void Info() {
```

Comment:
```suggestion
	public void displayInfo() {
```
tên method nên bắt đầu với động từ và mô tả dc nhiệm vụ của method đó nhé